connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -923,7 +928,7 @@ object SparkSession extends Logging {
      */
     def create(): SparkSession = {
       val session = tryCreateSessionFromClient()
-        .getOrElse(SparkSession.this.create(builder.configuration))
+        .getOrElse(SparkSession.this.create(getClientConfiguration()))
       setDefaultAndActiveSession(session)
       applyOptions(session)
       session
```

Comment:
Orthogonal: Why are create and getOrCreate functions exactly the same, while I think they should be doing something slightly different??

Reply:
`getOrCreate()` will try to retrieve cached SparkSession if exists. `create()` only creates a new SparkSession

Reply:
:facepalm: I'm blind and didn't notice the `SparkSession.this.create` vs `sessions.get`